BUILD.gn

```diff
@@ -376,6 +389,7 @@ if (current_toolchain != "${dir_pw_toolchain}/default:default") {
 
   if (enable_fake_tests) {
     chip_build("fake_platform") {
+      test_group = "//src:fake_platform_tests"
```

Comment:
@cecille FYI per offline discussion, full test run with fake platform can be removed.